content/docs/studio/python-api.md

```diff
@@ -0,0 +1,35 @@
+# Python API
+
+The purpose of Studio Python API is to give programmatic access to information
+in Studio and executing actions in it.
+
+We provide Studio Python API as a part of DVC python package. To use it, you
+need to [install](/doc/install) DVC with `pip` or `conda`.
+
+This reference provides the details about the functions in the `dvc.studio`
+module, which can be loaded in any regular way, for example:
+
+```py
+import dvc.studio
```

Comment:
Since it's under the dvc namespace, should we put it in https://dvc.org/doc/api-reference and then link between here and the rest api page?

Reply:
Added a link from DVC Python API page to Studio Python API page.